.github/workflows/pullrequest-stale.yml

```diff
@@ -0,0 +1,25 @@
+name: Stale
+on:
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  stale:
+    if: github.repository_owner == 'ls1intum'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 7 days stale PRs
+        uses: actions/stale@v5
+        with:
+          days-before-stale: 7
+          days-before-close: 14
+          # Disable issue checking, only PR
+          days-before-issue-stale: -1
+          remove-stale-when-updated: true
+          stale-pr-label: "stale"
+          exempt-pr-labels: "no-stale"
```

Comment:
These labels currently do not exist for the Artemis repository.
It might make sense to create them beforehand so you can directly give them proper coloring.

Also, adding some developer documentation about this feature would be a good idea, especially about when to (not) use `no-stale`.

Reply:
Here is the lable, what do you think about the color?
![image](https://user-images.githubusercontent.com/1368405/182637723-2590b2ca-ab1a-4ef6-a634-eac9b053f1e2.png)

Should the documentation added to the `CONTRIBUTOR GUIDE` section in the docs?

Reply:
Yes, somewhere in this section would be a good idea:
https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process/#development-process-how-to-merge-changes